src/main/resources/ca/corefacility/bioinformatics/irida/model/enums/workflows/MLSTMentalist/0.1.3/irida_workflow.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <iridaWorkflow>
-    <id>beee0266-c141-45ef-8d3e-9c471a90de88</id>
```

Comment:
Could you leave the `id` as it is for the older version.  This is stored in our database and so updating would break any previously-run results.  Changing the `version` is okay though.

Reply:
The uuid used by IRIDA is the one on the `*.xml` file.  The uuid in the Galaxy workflow is unused by IRIDA and can safely be ignored.  Thanks for pointing this out.